pkg/asset/installconfig/baremetal/baremetal.go

```diff
@@ -3,10 +3,11 @@ package baremetal
 
 import (
 	"fmt"
-
 	"github.com/AlecAivazis/survey/v2"
 	"github.com/AlecAivazis/survey/v2/terminal"
+	"github.com/openshift/installer/pkg/types/defaults"
 	"github.com/pkg/errors"
+	"net"
```

Comment:
Imports must be in 3 groups: stdlib, third-party, and local. This is enforced by the gofmt test:

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_installer/7398/pull-ci-openshift-installer-master-gofmt/1731783442779082752#1:build-log.txt%3A34-48